bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st

```diff
@@ -6,9 +6,5 @@ prepareBootstrap
 		exportKernelPackageNames;
 		exportAllPackagesIntoMcz;
 		exportMonticelloInStFile;
-		exportMultilingualInStFile;
-		exportSUnitInStFile;
-		exportCoreTestsInStFile;
 		exportFileSystemInStFile;
-		exportDeprecatedInStFile;
-		exportJenkinsToolsInStFile.
```

Comment:
Are these not used anymore?

Reply:
The Jenkins tools are loaded by Hermes, and the deprecated file system is still used. It is in the next line of the diff.